.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,17 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
PyCharm should prompt you to turn this inspection on when it detects `__future__` imports. Also, switching to this branch with an existing project didn't turn the inspection on for me because PyCharm defaults to IDE level settings, not project.

![image](https://cloud.githubusercontent.com/assets/123745/23440261/aa310210-fde0-11e6-8a88-7d949679e94e.png)